src/elements/MetricsSelector/MetricsSelector.js

```diff
@@ -77,6 +87,20 @@ const MetricsSelector = ({ maxSelectionNumber, metrics, name, onSelect, preselec
     isOpen && 'metrics-selector-header_open'
   )
 
+  useEffect(() => {
+    if (!modelEndpointUid) return
+    selectedModelEndpointUid.current = modelEndpointUid
```

Comment:
you don't use modelEndpointUid. Delete this prop and ref

Reply:
done